### DISCUSSION TITLE
Secure setup file

### DISCUSSION THREAD

**Discussion by imajidtp** [postfixadmin/postfixadmin#723] [General]
Hello

Is it necessary to delete or … the setup.php file after installation? so that other people do not have access through the link?

**Comment by DavidGoodwin** [comment]
if you're using a really old version of Postfixadmin - yes ... but it's been a long time since then ... 

You won't need to delete setup.php for version 3.3.x 

(If setup.php is asking for a 'setup password' then no, there's no need to delete it)

**Reply by imajidtp** [reply]
I am using the latest version, thank you for your replay
